### DISCUSSION TITLE
Is self-hosting Plausible for me?

### DISCUSSION THREAD

**Discussion by perdittmann** [plausible/analytics#1071] [Self-Hosted Support] [answered]
Hey everyone,

I just stumbled across Plausibe and I am intrigued. But looking closer, I am also a little confused.
So I am wondering if I am part of Plausible's target audience. Maybe you can help me.

I want a self-hosted solution. Self-hosted for me means being compatible with a shared hosting server, where all my clients (self-employed web designer and developer) can be expected to be. I can expect access to a PHP 7/MySQL setting. 
I only have a vague notion what Docker might be, and if I'd even be able to get it to run on a shared hosting server. Or on my own machine with an FTP access? You get the scope of my … well, knowledge.

I guess I'd need a "upload this to your server, set up a database and run this install script" kind of solution. Or intensive support. And probably different customers.

So if anyone could point me in the right direction – or tell me, move on, that's not for you – I'd be really grateful!
Many thanks in advance
Per

**Comment by Vigasaurus** [comment, marked as answer]
Plausible really won't work on shared hosting, it needs significant access between running two of its own databases and Docker - both of which are likely blocked on most every shared hosting solution. As far as running it on your owned hardware, that's possible, but probably more work than it's worth. 

My only suggestion for you if you do want analytics from Plausible is to use the cloud version - here you can have as many sites as you want only paying based on the total number of pageviews across all your sites (it's super easy and fast as opposed to self hosting, for most).

**Reply by perdittmann** [reply]
Thanks a lot. That is what I assumed, what a pity.

I am not a friend of cloud solutions, but I will look into that, maybe I can be convinced. Or I'll continue my quest for a simple and private analytics software. 😊
